server/locale/DE/GPL/styles/flex/ratio--6x1.css

```
@@ -0,0 +1,82 @@
+/* Not a valid style option ratio, but used as the mobile base for 8x1 and 20x1 */
+@media (min-aspect-ratio: 60/11) and (max-width: 501px) {
+    .message__headline {
+        font-size: 4vw;
+        line-height: 1.1em;
+        display: inline;
+    }
+    .message__disclaimer {
+        font-size: 2.5vw;
+    }
```

I think we can target tag--default instead here.
```suggestion
    .tag--default {
        font-size: 3.1vw;
    }
```